src/docs/sphinx/basicExamples/triaxialDriver/Example.rst

```diff
@@ -0,0 +1,263 @@
+ 
++--------------------+-------------------------+--------------------------+---------------------------+
+| **mode**           | **axial loading**       | **radial loading**       | **initial stress**        |
++--------------------+-------------------------+--------------------------+---------------------------+
+| ``strainControl``  | axial strain controlled | radial strain controlled | isotropic stress using    |
+|                    | with ``axialControl``   | with ``radialControl``   | ``initialStress``         |
++--------------------+-------------------------+--------------------------+---------------------------+
+| ``stressControl``  | axial stress controlled | radial stress controlled | isotropic stress using    |
+|                    | with ``axialControl``   | with ``radialControl``   | ``initialStress``         |
++--------------------+-------------------------+--------------------------+---------------------------+
+| ``mixedControl``   | axial strain controlled | radial stress controlled | isotropic stress using    |
+|                    | with ``axialControl``   | with ``radialControl``   | ``initialStress``         |
++--------------------+-------------------------+--------------------------+---------------------------+
+
+A triaxial test is usually conducted on a confined rock sample to determine material properties. 
+As shown, a conventional triaxial test is described using the ``mode="mixedControl"`` testing mode in the ``TriaxialDriver``.
+
+In a triaxial test, the testing sample is under confining pressure (radial stresses) and subject to increased axial load. Therefore, a stress control ``radialControl="stressFunction"`` is defined in the radial direction to impose confining pressure. A strain control ``axialControl="strainFunction"`` is applied in the axial direction to represent axial compression.
+ 
+The initial stress is specified by ``initialStress="-10.e6"``. To ensure static equilibrium at the first timestep, its value should be consistent with the initial set of applied stresses defined in axial or radial loading functions.
+This stress has a negative value due to the negative sign convention for compressive stress in GEOSX.
+
+
+Then, ``steps="200"`` defines the number of load steps and ``output="simulationResults.txt"`` specifies an output file to which the simulation results will be written. 
+
+
+.. literalinclude:: ../../../../../inputFiles/solidMechanics/triaxialDriver_ExtendedDruckerPrager.xml
+    :language: xml
+    :start-after: <!-- SPHINX_TASK -->
+    :end-before: <!-- SPHINX_TASK_END -->
+
+
+In addition to triaxial tests, volumetric and oedometer tests can be simulated by changing the ``strainControl`` mode, and by defining loading controls in axial and radial direction accordingly. A volumetric test can be modelled by setting the axial and radial control functions to the same strain function, whereas an oedometer test runs by setting the radial strain to zero.
```

Comment:
Maybe link to triaxialDriver documentation here

Reply:
Add it